src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -73,8 +73,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                        help='Default action to apply when no rule matches. Only applicable to Premium SKU.')
 
     with self.argument_context('acr import') as c:
-        c.argument('source_image', options_list=['--source'], help="The source identifier will be either a source image name or a fully qualified source.")
-        c.argument('source_registry', options_list=['--registry', '-r'], help='The source container registry can be name, login server or resource ID of the source registry.')
+        c.argument('source_image', options_list=['--source'], help="The source identifier. If --registry is not specified, this parameter must be a fully qualified source (contains login server). If source registry is specified with --registry, this parameter must be a source image name (not contain login server).")
```

Comment:
I don't think the last sentence " This is because a valid source image name can contain a login server name." is necessary, as it does not help in using this command and may add confusion.

Reply:
@Wwwsylvia, You're right, it could cause confusion. Sounds good!